abb_experimental.rosinstall

```diff
@@ -0,0 +1,5 @@
+#Temporary until PR accepted by https://github.com/ros-industrial/
+- git: {local-name: abb, uri: 'https://github.com/AustinDeric/abb.git', version: kinetic-devel}
```

Comment:
Personally I'm really not comfortable telling users in the official readme of `abb_experimental` to build a fork of `abb`. Would it be really a problem if we wait with this PR until your PR against `ros-industrial/abb` is merged?